tests/test_feature_validation.py

```diff
@@ -3,43 +3,61 @@
 import pytest
 
 from pyam import IamDataFrame, validate, categorize, require_variable
-from pyam.utils import META_IDX
+from pyam.utils import IAMC_IDX, META_IDX
+
+from .conftest import TEST_YEARS
+
+
+# add row for `Primary Energy|Gas` to test that all permutations
+DATA_GAS = pd.DataFrame(
+    [
+        ["model_a", "scen_a", "World", "Primary Energy|Gas", "EJ/yr", 0.5, 3],
+        ["model_a", "scen_b", "World", "Primary Energy|Gas", "EJ/yr", 1, 2],
+    ],
+    columns=IAMC_IDX + TEST_YEARS,
+)
 
 
 @pytest.mark.parametrize(
     "kwargs",
     (
         dict(),
         dict(variable="Primary Energy"),
+        dict(variable="Primary Energy", year=[2005, 2010]),
```

Comment:
Is there a specific reason you're using `dict()` over `{}`?
```suggestion
        {"variable": "Primary Energy", "year": [2005, 2010]},
```

Reply:
I think readability is better thanks to the syntax-highlighting...

Reply:
Fair